main.py

```diff
@@ -1,10 +1,45 @@
+from django.db import transaction
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
     pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+    with transaction.atomic():
+        for nickname, player_data in players_data.items():
+            race_data = player_data["race"]
+            race, _ = Race.objects.get_or_create(
+                name=race_data["name"],
+                defaults={"description": race_data.get("description", "")}
+            )
+
+            for skill_data in race_data.get("skills", []):
+                Skill.objects.get_or_create(
+                    name=skill_data["name"],
+                    race=race,
+                    defaults={"bonus": skill_data["bonus"]}
```

Comment:
Ensure that the 'bonus' field in the Skill model is correctly defined as a CharField if it represents text. If it represents a numerical value, consider changing it to a more appropriate field type in the model definition.